### DISCUSSION TITLE
Android prior to 7 and certificates in 2021

### DISCUSSION THREAD

**Discussion by Mladen13QA** [vercel/vercel#6421] [Help]
Is there a workaround for Android devices prior to 7 not being able to connect to Vercel's HTTPS servers?

**Comment by majidkhan12** [comment]
Yes[, ](https://apktiger.com/american-dad-apocalypse-soon-mod-apk/)there is a place for android devices.

**Reply by Marija321** [reply]
What does that mean that there is a place for android devices? Does it relate to this question?
